MANIFEST.SKIP

```diff
@@ -4,6 +4,8 @@
 ,v$
 \B\.svn\b
 \B\.cvsignore$
+\B\.git\b
```

Comment:
The manifest.skip file should be removed.
It is only needed for CPAN and should be auto-generated.